src/main/java/net/coderbot/iris/gui/element/ShaderPackListWidget.java

```diff
@@ -162,4 +186,38 @@ public void render(MatrixStack matrices, int index, int y, int x, int entryWidth
 			drawCenteredText(matrices, MinecraftClient.getInstance().textRenderer, label, (x + entryWidth / 2) - 2, y + (entryHeight - 11) / 2, 0xC2C2C2);
 		}
 	}
+
+	public static class EnableShadersButtonEntry extends BaseEntry {
+		public boolean enabled;
+
+		public EnableShadersButtonEntry(boolean enabled) {
+			this.enabled = enabled;
+		}
+
+		@Override
+		public void render(MatrixStack matrices, int index, int y, int x, int entryWidth, int entryHeight, int mouseX, int mouseY, boolean hovered, float tickDelta) {
+			GuiUtil.bindTexture(GuiUtil.IRIS_WIDGETS_TEX);
+			RenderSystem.color4f(1.0f, 1.0f, 1.0f, 1.0f);
+			RenderSystem.enableBlend();
+			RenderSystem.enableTexture();
```

Comment:
If your goal is to abstract things, these need to be in GuiUtil too. Perhaps this should just be part of drawButton.

Reply:
Will do.